### PR TITLE
Issue #104: Endpoint to search labels

### LINKED ISSUE #104: As a dev, I want to be able to retrieve a list of labels with some infos about each labels.
## Description  

**Context**
We want to display a list of labels to the user on the frontend

**Problem Statement**
At the moment there is no way to know what are the saved labels in the database unless you already know their ids. 
It would be nice to have a route to ask for the labels `kth-nth` with some informations about them.
The required informations would be :
- Label ID
- Company name
- Fertiliser Name
- Registred No
- Lot Number
- User
- Upload Date

**Acceptance Criteria**
[Provide clear criteria for what needs to be done to  
solve the problem and close the issue. This helps developers understand the  
desired outcome.]  

- [ ] A route is available to get a list of labels 
- [ ] The route asks for which elements you want to retrieve (would be a kth element to nth element)
- [ ] The route answers with a list of n-k labels 
- [ ] Each labels contains the elements previously written

**Additional Information** 
see ai-cfia/fertiscan-frontend#176

### PR DESCRIPTION
**Pull request by snakedye**
Closes #104 

This is the search query so far:
```
{
	keywords: [],
	label_id: "",
	user_id: "",
	registration_number: ""
}
```

I didn't review it in depth feel free to give me feedback @Youllou @Francois-Werbrouck

### CODE REVIEW

**Comment by snakedye** [comment]
I think we just just use `GET /form` instead of creating another route. This would avoid making too many requests to the database.

**Comment by Francois-Werbrouck** [comment]
What is the purpose of the keyword array?

**Comment by snakedye** [comment]
@Francois-Werbrouck It was to supposed to be some words somewhere in the form but it was scraped in the last meeting. We'll start with one. Since on [PR #77](https://github.com/ai-cfia/ailab-datastore/tree/77-create-an-inspection-from-a-json-form) there's [get_all_user_inspection](https://github.com/ai-cfia/ailab-datastore/blob/075f7238135a02d447361d82e82f3390a99853a1/datastore/db/queries/inspection/__init__.py#L133) I'll use the user_id as the sole paramater.

File: app.py [hunk 1]
```diff
@@ -129,13 +131,18 @@ def discard_form(form_id):
...
-    query = request.json
-    if query is None:
-        return jsonify(error="Missing search prompt!"), HTTPStatus.BAD_REQUEST
-    search_query = SearchQuery(**query)
+    # Database cursor
+    cursor = conn.cursor()
+
+    user_id = request.args.get('user_id')
+    if user_id is None:
+        return jsonify(error="Missing username!"), HTTPStatus.BAD_REQUEST
```
Comment (Francois-Werbrouck) [inline app.py:139]: Should be user not registered or something better representing the issue. We aren't using Username so far, we use email address

File: requirements.txt [hunk 2]
```diff
@@ -6,4 +6,4 @@ flasgger==0.9.7.1
 # Add the fertiscan-pipeline package from GitHub
 git+https://github.com/ai-cfia/fertiscan-pipeline.git@main
 # Add the datastore to interact with the database
-git+https://github.com/ai-cfia/nachet-datastore.git@77-create-inspection-fixes
+git+https://github.com/ai-cfia/nachet-datastore.git@77-create-an-inspection-from-a-json-form
```
Comment (Francois-Werbrouck) [inline requirements.txt:9]: Should soon be main!

**Comment by snakedye** [comment]
Merge into https://github.com/ai-cfia/fertiscan-backend/pull/106

**Pull request closed without merging**